src/data/equiv/module.lean

```diff
@@ -5,6 +5,7 @@ Authors: Nathaniel Thomas, Jeremy Avigad, Johannes Hölzl, Mario Carneiro, Anne
   Frédéric Dupuis, Heather Macbeth
 -/
 import algebra.module.linear_map
+import linear_algebra.basic
```

Comment:
This almost certainly won't work.

Can you instead just move `add_equiv.to_linear_equiv` into this file?

Reply:
Hmm, vscode didn’t look unhappy, but maybe I was misreading it’s status. At some point I got an error that it can’t resolve an import, but then it disappeared again.

Will try your suggestion.

Reply:
That is, move these lines:

https://github.com/leanprover-community/mathlib/blob/201aeaa3fd98e7dd6945bf38236af4449be2275f/src/linear_algebra/basic.lean#L2537-L2552